### PR TITLE
feat: events sufficient for indexer in MangroveOrder

### PR DESCRIPTION
**Pull request by peterMangrove**

### CODE REVIEW

**Comment by peterMangrove** [comment]
Did not use specific uint's like `uni96`, because `TakerOrder` and `TakerOrderResult` did not specify the types. We can make this change later, when we have a large review of our Events.

File: src/strategies/interfaces/IOrderLogic.sol [hunk 1]
```diff
@@ -58,19 +58,34 @@ interface IOrderLogic {
   ///@param outbound_tkn The outbound token of the order.
   ///@param inbound_tkn The inbound token of the order.
   ///@param taker The address of the taker
+  ///@param fillOrKill The fillOrKill that take was call with
```
Comment (lnist) [inline src/strategies/interfaces/IOrderLogic.sol]:
```suggestion
  ///@param fillOrKill The fillOrKill that take was called with
```

Comment (lnist) [reply src/strategies/interfaces/IOrderLogic.sol]: Same for expiry, restingOrder

**Review by lnist — CHANGES_REQUESTED** [review]

File: src/strategies/interfaces/IOrderLogic.sol (hunk 1 above)
Comment (peterMangrove) [reply src/strategies/interfaces/IOrderLogic.sol]: Done

**Review by lnist — APPROVED** [review]

File: src/strategies/interfaces/IOrderLogic.sol [hunk 2]
```diff
@@ -58,19 +58,34 @@ interface IOrderLogic {
...
   ///@param fillWants If true, the market order stopped when `takerWants` units of `outbound_tkn` had been obtained; otherwise, the market order stopped when `takerGives` units of `inbound_tkn` had been sold.
+  ///@param restingOrder The restingOrder boolean take was called with
+  ///@param expiryDate The expiry date take was called with
   ///@param takerGot How much the taker got
   ///@param takerGave How much the taker gave
-  ///@param penalty How much penalty was given
+  ///@param bounty How much bounty was given
+  ///@param fee How much fee was paid for the order
+  ///@param restingOrderId If a restingOrder was posted, then this holds the offerId for the restingOrder
+
```
Comment (espendk) [inline src/strategies/interfaces/IOrderLogic.sol]: Blank line should be removed

```suggestion
```

**Review by espendk — CHANGES_REQUESTED** [review]

**Review by espendk — APPROVED** [review]

**Comment by espendk** [comment]
> Did not use specific uint's like `uni96`, because `TakerOrder` and `TakerOrderResult` did not specify the types. We can make this change later, when we have a large review of our Events.

It might even be that it's cheaper gas wise to keep `uint` and not make the types smaller: https://ethereum.stackexchange.com/questions/3067/why-does-uint8-cost-more-gas-than-uint256

I agree with Peter that we should push this to the later discussion and get this PR merged.

**Review by jkrivine — APPROVED** [review]

**Pull request merged**